Balanced Paranthesis/balancedparanthesis.js

```diff
@@ -0,0 +1,15 @@
+ function balanced_paranthesis( brackets ){
+ 
+let count=0;
+   
+   for(let i = 0 ; i < brackets.length ; ++i)
+
```

Comment:
Here if and else should be enclosed inside a block or else you will get an exception saying else without a block.

Reply:
I mentioned to add to for block not for if and else. I have explained the cause for not enclosing for loop in block. Please add that.